README.md

```diff
@@ -1541,6 +1547,196 @@ support preserving stats between role reload
 (see [tarantool/metrics#334](https://github.com/tarantool/metrics/issues/334)),
 thus this feature will be unsupported for `metrics` driver.
 
+### Read view
+
+A read view is an in-memory snapshot of data on instances that isn’t affected by future data modifications. Read views allow you to retrieve data using the `read_view_object:select()` and `read_view_object:pairs()` operations.
+
+Read views can be used to make complex analytical queries. This reduces the load on the main database and improves RPS for a single Tarantool instance.
+
+Read views have the following limitations:
+
+  * Only the memtx engine is supported.
+  * Read view can be used starting from Tarantool Enterprise v2.11.0.
+
+#### Creating a read view
+
+To create a read view, call the `crud.readview()` function.
+
+```lua
+local foo = crud.readview(opts)
+```
+
+where:
+
+* `opts`:
+  * `name` (`?string`) - name of the read view
+  * `timeout` (`?number`) - `vshard.call` timeout (in seconds)
+
+**Example:**
+
+```lua
+local foo = crud.readview({name = 'foo', timeout = 3})
+```
+
+#### Closing a read view
+
+When a read view is no longer needed, close it using the `read_view_object:close()` method because a read view may consume a substantial amount of memory.
+
+```lua
+local foo = foo.readview()
+foo:close(opts)
+```
+
+where:
+
+* `opts`:
+  * `timeout` (`?number`) - `vshard.call` timeout (in seconds)
+
+Otherwise, a read view is closed implicitly when the read view object is collected by the Lua garbage collector.
+
+**Example:**
+
+```lua
+local foo = crud.readview()
+foo:close({timeout = 3})
+```
+
+#### Read view select
+
+`read_view_object:select()` supports multi-conditional selects, treating a cluster as a single space, same as `crud.select`.
+
+```lua
+local foo = crud.readview()
+local objects, err = foo:select(space_name, conditions, opts)
+foo:close()
+```
+
+where:
+
+* `space_name` (`string`) - name of the space
+* `conditions` (`?table`) - array of [select conditions](#select-conditions)
+* `opts`:
```

Comment:
Let's better make a link to select opts and list which options are not supported here.